packages/lit-dev-content/rollup.config.js

```diff
@@ -100,14 +99,13 @@ export default [
         showMinifiedSize: false,
       }),
     ],
+    preserveEntrySignatures: 'strict',
```

Comment:
In the PR comment you wrote: `preserveEntrySignatures: false` while here it is set to `strict`. I assume we need it explicitly set to generate the chunk?

Reply:
ah, good catch. I think it should be `false` as the docs and the warning message states `false` when building for a web app vs `strict` when building for a library.

Reply:
so setting it to `'strict'` which was the behavior when it wasn't set results in `js/global/hydrate-common-components.js` have `import '../hydrate-common-components.js` causing another request to happen. 
<img width="661" alt="image" src="https://github.com/lit/lit.dev/assets/40413829/e2c1ff6c-e097-48d9-a404-b2470e55d8b4">

with `false`, it doesn't create that extra step and the needed code is in `js/global/hydrate-common-components.js`.
<img width="757" alt="image" src="https://github.com/lit/lit.dev/assets/40413829/244a4daa-7cf1-405f-8661-f347ea572ba6">

Reply:
Awesome! Confirmed in preview URL that we now have one less request! Great removal of a unnecessary network round trip.

Comparing 
https://pr1136-037cc06---lit-dev-5ftespv5na-uc.a.run.app/js/global/hydrate-common-components.js to 
https://lit.dev/js/global/hydrate-common-components.js

So much better!